proposals/lip-0061.md

```diff
@@ -664,7 +664,7 @@ The function returns a valid certificate object that can be submitted to blockch
 For readability, we describe the logic using two functions, the main function `getNextCertificateFromAggregateCommits` and the auxiliary function `checkChainOfTrust`.
 
 ```python
-def getNextCertificateFromAggregateCommits(lastCertifiedHeight: uint32) -> Certificate:
+def getNextCertificateFromAggregateCommits(lastCertifiedHeight: uint32) -> Certificate | None:
```

Comment:
We should do the same for the function `getNextCertificateFromSingleCommits`.

Reply:
done